routes/borrowingsRouter.js

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import {borrowBook, returnBook, viewAllBorrowedBooks, viewAllBorrowingsByMember} from "../controllers/borrowingsController.js";
+
+const borrowingsRouter = express.Router();
+
+// Borrow book
+borrowingsRouter.post("/", borrowBook);
+
+// Return book
+borrowingsRouter.put("/", returnBook);
```

Comment:
I believe here for return book is updating the state of the book in the `DB`; with that in mind, I think it would be better to take the `book id` in the params like `put("/{id})`.